Resources/public/js/views/fields/ez-dateandtime-view.js

```diff
@@ -48,24 +48,44 @@ YUI.add('ez-dateandtime-view', function (Y) {
          * @return String
          */
         _formatDate: function (date) {
-            return date.toLocaleDateString();
```

Comment:
that the same case as for `_formatTime`, this one should be deprecated and ez-date-view should redefine a `_formatDate` method.